<?xml version="1.0"?>
<launch>

  <!-- <arg name="nodelet_manager" default="velodyne_nodelet_manager"/> -->

  <!-- filter -->
<!--   <node pkg="nodelet" type="nodelet" name="passthrough_sensor_nodelet" args="load pcl/PassThrough $(arg nodelet_manager)" output="screen">
    <remap from="/passthrough_sensor_nodelet/input" to="/velodyne_points"/>
    <remap from="/passthrough_sensor_nodelet/output" to="/filtered_points"/>
    <param name="filter_field_name" value="z"/>
    <param name="filter_limit_min" value="-0.2"/>
    <param name="filter_limit_max" value="1.6"/>
    <param name="filter_limit_negative" value="false"/>
  </node> -->

  <!-- map -->
<!--   <node pkg="octomap_server" type="octomap_server_node" name="map_server">
    <remap from="/cloud_in" to="/globalmap"/>
    <remap from="/projected_map" to="/map"/>
    <param name="frame_id" value="map"/>
    <param name="resolution" value="0.1"/>
    <param name="pointcloud_min_z" value="0.4"/>
    <param name="pointcloud_max_z" value="1.8"/>
  </node>  -->

  <!-- move_base -->
  <node pkg="move_base" type="move_base" name="move_base" output="screen">
    <rosparam file="$(find fcsc)/params/move_base/real/move_base_params.yaml" command="load"/>
    <rosparam file="$(find fcsc)/params/move_base/real/global_costmap_params.yaml" command="load"/>
    <rosparam file="$(find fcsc)/params/move_base/real/local_costmap_params.yaml" command="load"/>
    <rosparam file="$(find fcsc)/params/move_base/real/global_planner_params.yaml" command="load"/>
    <rosparam file="$(find fcsc)/params/move_base/real/dwa_local_planner_params.yaml" command="load"/>

    <remap from="/cmd_vel" to="/rover_twist"/>
  </node>

</launch>
